Formula/mercurial.rb

```diff
@@ -13,6 +14,14 @@ class Mercurial < Formula
     sha256 "8934a8fc26c36e56f017d99ac43a0ebe9d6922f8276c1dfccd9ccb980ae05e45" => :mavericks
   end
 
+  option "with-custom-python", "Install against the python in PATH instead of "\
```

Comment:
I'd really prefer we didn't do this with options. Can we either EOS them, i.e.:
```ruby
  option "with-custom-python", <<-EOS.undent
    Install against the python in PATH instead of Homebrew's python
  EOS
```
or keep them on one line. It's possibly just me who finds `\` grim, but I'm unconvinced it's all that readable at a glance.